config-release/values-schema.yml

```diff
@@ -5,7 +5,7 @@
 #@schema/desc "Configuration explicitly for developing kapp-controller"
 dev:
   #@schema/desc "Location of kapp-controller image"
-  image_repo: ghcr.io/carvel-dev/kapp-controller
+  image_repo: ghcr.io/rcmadhankumar/kapp-controller
```

Comment:
I think this was intended for testing on your fork?

Reply:
Yes. this will be changed. will resolve this once the change is done.